auto_submit/test/requests/check_pull_request_test.dart

```diff
@@ -152,16 +157,16 @@ void main() {
       githubService.successMergeData = successMergeMock;
       githubService.createCommentData = createCommentMock;
       config = FakeConfig(githubService: githubService);
-      checkPullRequest = CheckPullRequest(config: config);
+      checkPullRequest = CheckPullRequest(config: config, pubsub: pubsub);
 
-      final Response response = await checkPullRequest.get(req);
+      final Response response = await checkPullRequest.get();
       final String resBody = await response.readAsString();
       expect(resBody, 'Remove the autosubmit label.');
     });
 
     test('Removes the label if non member does not have at least 2 member reviews', () async {
-      String webHookEvent = generateWebhookEvent(authorAssociation: '');
-      req = Request('GET', Uri.parse('http://localhost/'), body: webHookEvent);
+      PullRequest pullRequest = generatePullRequest(authorAssociation: '');
+      pubsub.publish('test-topic', pullRequest);
```

Comment:
If 'test-topic' is used in multiple places, we should make it into a const for the test

Reply:
Sure, added a const for the test topic.